### PR TITLE
Add changes to ballerina streams example & remove redundant changes

### PR DESCRIPTION
**Pull request by mohanvive**

### CODE REVIEW

File: examples/simple-filtering/simple_filtering.bal [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+type Child record {
+    string name;
+    int age;
+    string city;
+};
+
+
+int index = 0;
+
+// Streams which are based on above-created constraint types.
```
Comment (samgnaniah) [inline examples/simple-filtering/simple_filtering.bal]: Change to "Streams that are based on the constraint types created above."

File: examples/simple-filtering/simple_filtering.bal [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+stream<Person> personStream = new;
+stream<Child> childrenStream = new;
+
+Child[] globalChildrenArray = [];
+
+function initFilterQuery() {
+    // Streaming query which filter events based on the attribute age.
```
Comment (samgnaniah) [inline examples/simple-filtering/simple_filtering.bal]: // Streaming query that filters events based on the attribute age.

File: examples/simple-filtering/simple_filtering.bal [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    // Filterted events are pushed to a stream called `childrenStream`.
```
Comment (samgnaniah) [inline examples/simple-filtering/simple_filtering.bal]: // Filtered events are pushed to a stream called `childrenStream`.

File: examples/simple-filtering/simple_filtering.bal [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+            foreach var c in children {
+                childrenStream.publish(c);
+            }
+        }
+    }
+}
+
+public function main() {
+
+    // Sample events that represent the different person.
```
Comment (samgnaniah) [inline examples/simple-filtering/simple_filtering.bal]: // Sample events that represent a different person.

File: examples/simple-filtering/simple_filtering.description [hunk 5]
```diff
@@ -0,0 +1,7 @@
+//Simple filtering can be performed for the events which come in the stream.
```
Comment (samgnaniah) [inline examples/simple-filtering/simple_filtering.description]: // Simple filtering can be performed for the events that come in the stream.
// In this example, events are related to the person being monitored and there 
// will be an alert triggered when there is a person whose age is less than or equal to 16.
// This example can be extended to make simple filtering based on some condition.
// Ballerina Streams is an experimental feature. Please use `--experimental` flag when 
// compiling Ballerina files that have streaming constructs.

File: examples/simple-filtering/simple_filtering.bal (hunk 1 above)
Comment (mohanvive) [reply examples/simple-filtering/simple_filtering.bal]: Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/12750.

File: examples/simple-filtering/simple_filtering.bal (hunk 2 above)
Comment (mohanvive) [reply examples/simple-filtering/simple_filtering.bal]: Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/12750.

File: examples/simple-filtering/simple_filtering.bal (hunk 4 above)
Comment (mohanvive) [reply examples/simple-filtering/simple_filtering.bal]: FIxed in https://github.com/ballerina-platform/ballerina-lang/pull/12750.

File: examples/simple-filtering/simple_filtering.bal (hunk 3 above)
Comment (mohanvive) [reply examples/simple-filtering/simple_filtering.bal]: Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/12750.

File: examples/simple-filtering/simple_filtering.description (hunk 5 above)
Comment (mohanvive) [reply examples/simple-filtering/simple_filtering.description]: Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/12750.

**Pull request merged**